docs/system-admin-guide/users-permissions/roles-permissions/README.md

```diff
@@ -77,13 +77,16 @@ To create a global role tick the box "Global Role" when [creating a new role](#c
 
 ![global-roles-in-openproject](image-20210308171607279.png)
 
-You can choose between these global roles:
+You can choose between these global permissions:
 
-- **Create project**: Assign this role to users to enable them to create new projects without being system administrator. 
-- **Create and edit users**: Assign this role to users who should be able to create or invite new users and edit their profiles in a limited way. 
-  Users with this role can see all users of your OpenProject instance and can add users and edit the name, username, email address and language of a user. They can't delete or lock users. They can only see and add users to projects where they have permissions to see project members.
+- **Create project**: Assign this permission to users to enable them to create new projects without being system administrator. 
+  You can change the [project role](../../../user-guide/members) a user with this permission is assigned in a newly created project [this way](../../system-settings/project-system-settings/#settings-for-new-projects).
+- **Create and edit users**: Assign this permission to users who should be able to create or invite new users and edit their profiles in a limited way. 
+  Users with this permission can see all users of your OpenProject instance and can add users and edit the name, username, email address and language of a user. They can't delete or lock users. 
+  They can only see the project membership of users for projects in which they have permission to see the members (e.g. as Project admin or Member). They can only manage project membership of users for projects in which they have permission to manage members (e.g. as Project admin).
   The user profile will look like this for them (user name and email address were redacted): ![create-and-edit-users-role](image-20210308180635158.png)
-- **Create, edit and delete placeholder users**: Assign this role to users (e.g. project admins) who should be able to manage [placeholder users](../placeholder-users). 
-  Users with this role can see all placeholder users in your OpenProject instance and can create, edit and delete placeholder users. They can only see and add placeholder users to projects where they have permissions to see project members.
+- **Create, edit and delete placeholder users**: Assign this permission to users (e.g. project admins) who should be able to manage [placeholder users](../placeholder-users). 
+  Users with this permission can see all placeholder users in your OpenProject instance and can create, edit and delete placeholder users.
+  They can only see the project membership of placeholder users for projects in which they have permission to see the members (e.g. as Project admin or Member). They can only manage project membership of placeholder users for projects in which they have permission to manage members (e.g. as Project admin).
```

Comment:
Maybe that sentence needs some "Hint:" or "Watch out:" as that sentence is meant to explain why placeholder users are not visible/editable **although** you have the right to create, edit, and delete them.